modules/tide_dashboard/config/install/views.view.tide_dashboard_my_content.yml

```diff
@@ -0,0 +1,1012 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_node_site
```

Comment:
field_node_site provided by `tide_site`, does this need to move to `optional` directory ?